test/json-sources.js

```diff
@@ -8,6 +8,12341 @@
  * and used from karma-setup.js.
  */
 window.JSONSources = {
+    "https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/samples/data/usdeur.json":
```

Comment:
This is probably not needed as we remove the stock demo and use our existing stock line